composer.json

```diff
@@ -6,7 +6,7 @@
     "require": {
         "php": ">=7",
         "ext-json": "*",
-        "keboola/juicer": "^4.9",
+        "keboola/juicer": "^4.9.1",
```

Comment:
jen bych to nechal na `^4.9` aby se to nemuselo přepisovat při každé "drobné" změně

Reply:
Rozdiel medzi `^4.9` a `^4.9.1` je iba v tom, ze:
 - `^4.9` vyzaduje minimalne `4.9.0`
 - `^4.9.1` vyzaduje minimalne `4.9.1`
 
 Teda presne to co chcem. 
 
 Operator `^` "zamyka" iba najvysiu uroven (ak verzia >= `1.0`)
https://getcomposer.org/doc/articles/versions.md#caret-version-range-